### ISSUE TITLE
[Terminal Chat AI] Add run button to execute suggested commands inline

### ISSUE THREAD

**Issue by g0t4** [microsoft/terminal#16435]
# Description of the new feature/enhancement

Add support to run generated commands right inside the chat and show the output inline?
 
- i.e. if I ask how to list markdown files, when the command is suggested I could click run (instead of copy) to list the files right inside the chat! 
  - or use action mapped to keyboard shortcut to run last suggested command
- command output would be available in subsequent chat context
  - so I could ask how to modify the `foo.md` file to replace `bar` with `boo`
  - verbose commands wouldn't work out well but that can be up to the end user to decide

TLDR, **interleave chat with command execution**. 

# Proposed technical implementation details (optional)

When commands are suggested the bubble could be a mini shell that would run the command. Could even allow for modifications before running. Then, show the output below, within the same bubble... once that runs further suggestions could be shown and/or the user could ask a follow up question.

Key details about profiles (i.e shell type/name) could be fed in as context to know what shells are available. Which would then allow suggesting/running commands in different shells/profiles.

**Comment by mcclure** [comment]
I like Windows Terminal, but adding Copilot/LLM integration would 100% convince me to stop using it.

**Comment by DHowett** [comment]
Closing in favor of Intelligent Terminal (which is a separate product, don't worry!)
